repos/c2sm/packages/py-scikit-build-core/package.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+# TODO (magdalena) copy from https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-scikit-build-core/package.py
+#   add new version (0.5.0)
+
+
+class PyScikitBuildCore(PythonPackage):
+    """scikit-build-core is a doubly improved build system generator
+    for CPython C/C++/Fortran/Cython extensions. It features several
+    improvements over the classic scikit-build build system generator."""
+
+    homepage = "https://github.com/scikit-build/scikit-build-core"
+    pypi = "scikit_build_core/scikit_build_core-0.2.0.tar.gz"
+    git = "https://github.com/scikit-build/scikit-build-core"
+
+    maintainers = ["wdconinc"]
+
+    version("0.2.0",
+            sha256=
+            "d2a76d9447a412038dc5e25dd259b03c25278661a0c7c3da766bb971c1a9acd2",
+            pypi=pypi)
```

Comment:
@dominichofer I get an error about this pypi option after the rebase. Do you know what has changed?

Reply:
No idea.